jcli/src/jcli_app/certificate/mod.rs

```diff
@@ -67,6 +68,16 @@ pub enum Error {
     TooManyPoolDelegations { actual: usize, max: usize },
     #[error("failed to build pool delegation")]
     InvalidPoolDelegation,
+    #[error("BlockDates should be consecutive, vote start ({vote_start}) cannot be bigger than vote end ({vote_end})")]
+    InvalidVotePlanVoteBlockDates {
+        vote_start: String,
```

Comment:
BlockDate already has a `Display` implementation, no need for the `String` here

Reply:
I did not saw any complicated type there so I decided to use plain `String`. I liked it more with the actual type anyway :)